helm/cluster/templates/clusterapi/_helpers_flatcar.tpl

```diff
@@ -263,9 +281,19 @@
     [Unit]
     Description=Teleport Service
     After=network.target
+    {{- if $.Values.providerIntegration.teleport.initialJoinToken }}
+    After=network.target teleport-init.service
+    Requires=teleport-init.service
+    RequiresMountsFor=/etc/teleport.yaml /etc/teleport-join-token /opt/teleport-node-role.sh
+    {{- end }}
```

Comment:
I _think_ it is valid to mention multiple `After`, but do you think it would be useful to combine them?

```suggestion
    {{- if $.Values.providerIntegration.teleport.initialJoinToken }}
    After=network.target teleport-init.service
    Requires=teleport-init.service
    RequiresMountsFor=/etc/teleport.yaml /etc/teleport-join-token /opt/teleport-node-role.sh
    {{- else }}
    After=network.target
    {{- end }}
```

Reply:
Totally agree with you, it will make it clearer.
committing suggestion